cookbooks/aws-parallelcluster-install/recipes/intel_mpi.rb

```diff
@@ -17,40 +17,46 @@
 
 return unless node['conditions']['intel_mpi_supported']
 
-intelmpi_modulefile = "/opt/intel/impi/#{node['cluster']['intelmpi']['version']}/intel64/modulefiles/intelmpi"
-intelmpi_installer_archive = "l_mpi_#{node['cluster']['intelmpi']['version']}.tgz"
-intelmpi_installer_path = "#{node['cluster']['sources_dir']}/#{intelmpi_installer_archive}"
-intelmpi_installer_url = "https://#{node['cluster']['region']}-aws-parallelcluster.s3.#{node['cluster']['region']}.#{aws_domain}/archives/impi/#{intelmpi_installer_archive}"
+intelmpi_installation_path = "/opt/intel/mpi/#{node['cluster']['intelmpi']['version']}"
+intelmpi_modulefile = "#{intelmpi_installation_path}/modulefiles/intelmpi"
+intelmpi_installer = "l_mpi_oneapi_p_#{node['cluster']['intelmpi']['full_version']}_offline.sh"
+intelmpi_installer_path = "#{node['cluster']['sources_dir']}/#{intelmpi_installer}"
+intelmpi_installer_url = "https://#{node['cluster']['region']}-aws-parallelcluster.s3.#{node['cluster']['region']}.#{aws_domain}/archives/impi/#{intelmpi_installer}"
```

Comment:
Here we are using the naming `impi`. Shouldn't it be `mpi`?

Reply:
Probably it's not worth it to rename it.

Reply:
To change it we need to move artifacts in a different S3 folder and we cannot move old ones so it's better to keep all IntelMPI artifacts together.